web/src/main/java/com/bazaarvoice/emodb/web/EmoService.java

```diff
@@ -339,6 +341,13 @@ private void evaluateMegabus()
         if (!runPerServiceMode(megabus)) {
             return;
         }
+
+        ResourceRegistry resources = _injector.getInstance(ResourceRegistry.class);
+        MegabusSource megabusSource = _injector.getInstance(MegabusSource.class);
+        resources.addResource(_cluster, "emodb-megabus-1", new MegabusResource1(megabusSource));
+
+        // No admin tasks are registered automatically in MEGABUS ServiceMode
+        _environment.admin().addTask(_injector.getInstance(LeaderServiceTask.class));
```

Comment:
Why is registering the `LeaderServiceTask` here necessary? The megabus already contains `            Aspect.task` as part of its service mode.

Reply:
yeah, updated.